### PR TITLE
Update Filter Feature in UG

### PR DESCRIPTION
**Pull request by jicsontoh**

### CODE REVIEW

**Review by ElginL — APPROVED** [review]
LGTM

**Review by notle1706 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -120,6 +121,11 @@ Examples:
 * `find fairprice` returns `NTUC Fairprice` <br>
 
 [//]: # (  ![result for 'find alex david']&#40;images/findAlexDavidResult.png&#41;)
+### Filtering the address book display : `filter`
+
+Coming Soon.
+
+### Deleting a person : `delete`
```
Comment (notle1706) [inline docs/UserGuide.md]: Perhaps you could remove this line before merging.

**Review by notle1706 — CHANGES_REQUESTED** [review]
Remove line 128 of UserGuide.md.

**Review by notle1706 — APPROVED** [review]

**Pull request merged**